examples/simple_regional/README.md

```diff
@@ -0,0 +1,17 @@
+# Simple Regional Cluster
```

Comment:
Since we're explicitly calling out that it's a regional example, lets delete `examples/simple` since that one one was inherently a regional cluster. I like it being `simple_regional` better... makes it more clear.